website/sidebars.json

```diff
@@ -669,7 +669,8 @@
 						"react-native/generate-web-preview-locally",
 						"react-native/web-preview-debug",
 						"react-native/expo-debug",
-						"react-native/flipper"
+						"react-native/flipper",
+						"react-native/performance-measurement"
```

Comment:
other new docs are missing on the sidenav bar

Reply:
The idea is to have the Performance Measurement as main page and other pages can be accessed through navigation.

Reply:
updated